cli_tools/common/utils/daisy/daisy_utils.go

```diff
@@ -45,15 +47,16 @@ var (
 		"windows-7-byol":      "windows/translate_windows_7_byol.wf.json",
 		"windows-10-byol":     "windows/translate_windows_10_byol.wf.json",
 	}
+	privacyRegex = regexp.MustCompile(`\[Privacy\->.*?<\-Privacy\]`)
 )
 
 // ValidateOS validates that osID is supported by Daisy image import
 func ValidateOS(osID string) error {
 	if osID == "" {
-		return fmt.Errorf("osID is empty")
+		return daisy.Errf("osID is empty")
 	}
 	if _, osValid := osChoices[osID]; !osValid {
-		return fmt.Errorf("os `%v` is invalid. Allowed values: %v", osID, reflect.ValueOf(osChoices).MapKeys())
+		return daisy.Errf(fmt.Sprintf("os `%v` is invalid. Allowed values: %v", osID, reflect.ValueOf(osChoices).MapKeys()))
```

Comment:
formatting here is, I assume, so that error gets logged to both personalized and anonymized log? 

Consider either documenting this trick or providing a separate method to log errors that are inherently anonymized?